packages/compile-solidity/compilerSupplier/loadingStrategies/LoadingStrategy.js

```diff
@@ -77,16 +77,22 @@ class LoadingStrategy {
     );
   }
 
+  markListeners() {
+    return new Set(process.listeners("uncaughtException"));
```

Comment:
Note, the current version of soljson listens for `unhandledRejection`. The logic in the rest of this PR can be extended to consider both handlers.

```suggestion
    return {
      uncaughtException: new Set(process.listeners("uncaughtException")),
      unhandledRejection: new Set(process.listeners("unhandledRejection""))
    };
```